cache_pdks/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Install PDKs.
+
+inputs:
+  pdk_root:
+    description: PDK root to install files.
+    required: true
+  ol_path:
+    description: OpenLane path.
+    default: .
+  ol_tag:
+    description: OpenLane docker tag.
```

Comment:
Curious if we could make this more reusable (outside of an openlane context) by specifying the open_pdks version instead?

Reply:
I could refactor and create a generic version of this action and then an OL specific wrapper. I wanted to keep this as straightforward as possible for OL users.

Reply:
If this action also makes the checkout of OL then there would OL would be on two paths. I have renamed the action to `openlane_cache_pdk` and removed `ol_path`, thus requiring OL to be already checked out.